server/routes/address.js

```diff
@@ -57,6 +57,17 @@ module.exports = ({ licenceService, nomisPushService }) => (router, audited, { p
       const { enterAlternative, bookingId } = req.body
       const { licence } = res.locals.licence
 
+      const errorObject =
+        licenceService.validateForm({
```

Comment:
can we add a route test for this too?

Reply:
I have assumed you mean a GEB test to check that the user lis taken from http://localhost:3000/hdc/taskList/{bookingId}  to server/routes/address.js if the RO has rejected the address??

Reply:
Just a normal Jest route test, e.g: risk.test.js or any of the ones in here: 
`test/routes`.

Reply:
done